### PR TITLE
feat: [STO-6029]: Updated Fossa Icon

### PR DESCRIPTION
**Pull request by jsentongo**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

**Comment by jsentongo** [comment]
retrigger titlecheck

File: packages/icons/src/fossa-inverse.svg
Comment (ishantmehtaharness) [inline packages/icons/src/fossa-inverse.svg:1]: are these 2 icons same/duplicate?

Comment (jsentongo) [reply packages/icons/src/fossa-inverse.svg:1]: Yeah they have to be as they're the same for both variations (shown in the screenshot)

**Review by ishantmehtaharness — APPROVED** [review]

File: packages/icons/src/fossa.svg [hunk 1]
```diff
@@ -1,5 +1,191 @@
...
-    </g>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" width="100%" viewBox="0 0 286 284" enable-background="new 0 0 286 284" xml:space="preserve">
```
Comment (knagurski) [inline packages/icons/src/fossa.svg:1]: The `width` needs to be removed along with `x` and `y`. I think you need to run these through a SVG optimiser (I like [ImageOptim](https://imageoptim.com/mac))

File: packages/icons/src/fossa-inverse.svg
Comment (ishantmehtaharness) [reply packages/icons/src/fossa-inverse.svg:1]: I agree with @knagurski 
We should not duplicate just for the same of name. We can re-export

Comment (jsentongo) [reply packages/icons/src/fossa-inverse.svg:1]: How do you re-export? Is there an example of this somewhere?

File: packages/icons/src/fossa.svg (hunk 1 above)
Comment (jsentongo) [reply packages/icons/src/fossa.svg:1]: Looking into the visuals.

File: packages/icons/src/fossa-inverse.svg
Comment (jsentongo) [reply packages/icons/src/fossa-inverse.svg:1]: When I plug 'import FossaInverse from './fossa.svg' into HarnessIcons.ts.. it gets stripped out as it's an auto generated file..

Comment (jsentongo) [reply packages/icons/src/fossa-inverse.svg:1]: This code runs for every Step Icon (in the pipeline studio) https://github.com/harness/harness-core-ui/blob/develop/src/modules/70-pipeline/components/PipelineDiagram/Nodes/DefaultNode/PipelineStepNode/PipelineStepNode.tsx#L70

It applies styling to invert the Step's icon if an inverse variant doesn't exist. For some of the Icons, this doesn't work so well and hence the need to make the inverse Icon (in uicore) then same at the normal variant. 

We don't explicitly call <Icon name='fossa' /> this is all taken care of by other components for all the steps. 
We just provide the icon name in the step (https://github.com/harness/harness-core-ui/blob/develop/src/modules/85-sto/components/PipelineSteps/BanditStep/BanditStep.tsx#L47)  and the inverting is done else where (the first link)

File: packages/icons/src/fossa.svg (hunk 1 above)
Comment (jsentongo) [reply packages/icons/src/fossa.svg]: Received sign-off from PM to go with this logo.

**Comment by jsentongo** [comment]
retrigger prettier

**Comment by jsentongo** [comment]
retrigger prettier

**Review by williamwissemann — APPROVED** [review]

**Pull request merged**